lib/actions/api.js

```diff
@@ -117,11 +117,21 @@ export function routingQuery(searchId = null, updateSearchInReducer = false) {
     // For multiple mode combinations, gather injected params from config/query.
     // Otherwise, inject nothing (rely on what's in current query) and perform
     // one iteration.
+    const { wheelchair: accessibleRoutingEnabled } = state.otp.currentQuery
     const iterations = state.otp.currentQuery.combinations
-      ? state.otp.currentQuery.combinations.map(({ mode, params }) => ({
-          mode,
-          ...params
-        }))
+      ? state.otp.currentQuery.combinations
+          .map(({ mode, params }) => ({
+            mode,
+            ...params
+          }))
+          // Remove non-accessible combinations when accessible routing is enabled
+          .filter((combination) => {
+            if (!accessibleRoutingEnabled) return true
+
+            // Explicit check for false is needed to avoid matching
+            // combinations without the pparameter set (undefined)
```

Comment:
Fix typo

Reply:
Good catch thanks! Fixed in 47595228122b9d50cefec2bddba18a25e8baa34e